src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-form/rgw-storage-class-form.component.html

```diff
@@ -0,0 +1,359 @@
+          type="type"
+          id="storage_class"
+          formControlName="storage_class"
+          [invalid]="
+            storageClassForm.controls.storage_class.invalid &&
+            storageClassForm.controls.storage_class.dirty
+          "
+        />
+      </cds-text-label>
+      <ng-template #storageError>
+        <span
+          class="invalid-feedback"
+          *ngIf="storageClassForm.showError('storage_class', formDir, 'required')"
+          i18n
+          >This field is required.</span
+        >
+      </ng-template>
+    </div>
+    <div class="form-item form-item-append"
+         cdsRow>
+      <div cdsCol>
+        <!-- Target Region -->
+        <cds-text-label
+          labelInputID="region"
+          i18n
+          [invalid]="
+            storageClassForm.controls.region.invalid && storageClassForm.controls.region.dirty
+          "
+          [invalidText]="regionError"
+          [helperText]="targetRegionText"
+          >Target Region
+          <input
+            cdsText
+            type="text"
+            id="region"
+            formControlName="region"
+            placeholder="e.g, us-east-1"
+            i18n-placeholder
+            [invalid]="
+              storageClassForm.controls.region.invalid && storageClassForm.controls.region.dirty
+            "
+          />
+        </cds-text-label>
+        <ng-template #regionError>
+          <span
+            class="invalid-feedback"
+            *ngIf="storageClassForm.showError('region', formDir, 'required')"
+            i18n
+            >This field is required.</span
+          >
+        </ng-template>
+      </div>
+      <div cdsCol>
+        <!-- Target Endpoint -->
+        <cds-text-label
+          labelInputID="endpoint"
+          i18n
+          [invalid]="
+            storageClassForm.controls.endpoint.invalid && storageClassForm.controls.endpoint.dirty
+          "
+          [invalidText]="endpointError"
+          [helperText]="targetEndpointText"
+          >Target Endpoint
+          <input
+            cdsText
+            type="text"
+            placeholder="e.g, http://ceph-node-00.com:80"
+            i18n-placeholder
+            id="endpoint"
+            formControlName="endpoint"
+            [invalid]="
+              storageClassForm.controls.endpoint.invalid && storageClassForm.controls.endpoint.dirty
+            "
+          />
+        </cds-text-label>
+        <ng-template #endpointError>
+          <span
+            class="invalid-feedback"
+            *ngIf="storageClassForm.showError('endpoint', formDir, 'required')"
+            i18n
+            >This field is required.</span
+          >
+        </ng-template>
+      </div>
+    </div>
+
+    <!-- Access Key  -->
+    <div class="form-item">
+      <div cdsCol
+           [columnNumbers]="{ md: 12 }"
+           class="d-flex">
+        <cds-password-label
+          labelInputID="access_key"
+          label="Target Access Key..."
```

Comment:
```suggestion
```
Actually there is no label prop https://github.com/carbon-design-system/carbon-components-angular/blob/cd447b6cf13310f650036572419c15f1b81e3470/src/input/password-input-label.component.ts#L135

Please correct other places as well.